docs/content/preview/deploy/multi-dc/async-replication.md

```diff
@@ -132,15 +132,17 @@ Replication lag is computed at the tablet level as follows:
 
 *hybrid_clock_time* is the hybrid clock timestamp on the source's tablet-server, and *last_read_hybrid_time* is the hybrid clock timestamp of the latest record pulled from the source.
 
-An example script [`determine_replication_lag.sh`](/files/determine_replication_lag.sh) calculates the replication lag. The script requires the [`jq`](https://stedolan.github.io/jq/) package.
+An example script `determine_replication_lag.sh` located in the `files` directory calculates the replication lag. The script requires the jq package.
```

Comment:
From [the slack thread](https://yugabyte.slack.com/archives/CHC5RSY74/p1650383646819669?thread_ts=1650380816.965229&cid=CHC5RSY74) it seems that this script does not work so we should remove any mention of it completely

Reply:
The ticket description says that `determine_repl_latency.sh` shoudl be removed. 

There's nothing about `determine_replication_lag.sh`

Reply:
Marko Rajcevic via slack:

I believe neither works
based off what I am seeing in the thread
Alex is under the impression its the same file but was just renamed
I just asked Tyler to confirm in the thread

Reply:
I removed any mentioning of these two scripts